### DISCUSSION TITLE
Troubleshooting page for common issues

### DISCUSSION THREAD

**Discussion by stilnat** [FirstGearGames/FishNet#762] [Documentation]
Hi, It'd be great to have a troubleshooting page for issues and errors and what to try to solve them. For instance, I got the following error yesterday : 
`Client encountered an error while parsing data for packetId ObjectSpawn. Message: Argument_InvalidOffLen.`
No idea what to do with it, Max kratt suggested it might be debug related, I removed the debug manager and that was it. Would never have found that on my own.

**Comment by FirstGearGames** [comment]
If you can provide 5 common problems here I'll be happy to look at making a page.

Please no tutorial requests, eg: how to use prediction with CC.

**Comment by FirstGearGames** [comment]
Added
